Source/WebKit/NetworkProcess/cocoa/NetworkProcessCocoa.mm

```diff
@@ -54,6 +54,11 @@
 #import <pal/spi/cocoa/NEFilterSourceSPI.h>
 #endif
 
+#if USE(EXTENSIONKIT) && __has_include(<WebKit-Swift.h>)
```

Comment:
Why? It seems like `USE(EXTENSIONKIT)` implies that we're compiling swift code and generating the Swift header.